File: 프로그래머스/Lv.1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.cs
using System.Collections.Generic;
using System.Linq;
public class Solution {
    public int[] solution(int[] arr) {
        int[] answer = new int[] {};
        List<int> list = arr.ToList();
        list.Sort();
        int min = list[list.Count-1];
        int minIdx =0;
        for(int i =0;i< arr.Length;i++)
        {
            if(min >= arr[i])
            {
                minIdx = i;
                min = arr[i];
            }
        }
        list = arr.ToList();
        list.RemoveAt(minIdx);
        if(list.Count == 0)
            list.Add(-1);
        answer = list.ToArray();
        return answer;
    }
}

File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.java
import java.util.Arrays;
class Solution {
    public int[] solution(int[] arr) {
        int[] answer = {-1};
            
        if(arr.length == 1)
            return answer;
        
        int min = Arrays.stream(arr)
            .min()
            .orElse(0);
        
        answer = Arrays.stream(arr)
            .filter(n -> n > min)
            .toArray();
        return answer;
    }
}